pkg/env/registry_mirroring.go

```diff
@@ -0,0 +1,6 @@
+package env
+
+var (
+	// RegistryMirroringEnabled enables processing registry mirrors during image enrichment.
+	RegistryMirroringEnabled = RegisterBooleanSetting("ROX_REGISTRY_MIRRORING_ENABLED", false)
```

Comment:
Do we want this to be a permanent setting or just a feature flag?

Reply:
Immediately act as 'feature flag' but if it sticks around long enough permanent 😉 . Will observe how the feature behaves over time.

Prefer not to use the `features` package due to that value not being changeable post release.

Reply:
if you are sure this is shipping in the next release, then this is good. if it misses, then we'll switch it to a feature flag